### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by aron-net**
Tribute page for Woldeab Woldemariam, also spelled Weldeab Weldemariam, was one of the original proponents of the Eritrean Independence movement and an uncompromising advocate of freedom considered by many Eritreans as the fathers of Eritrea nationalism.

### CODE REVIEW

**Review by ishimwezachee — COMMENTED** [review]
Hi @aron-helu , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no any review. You are supposed to review yourself and follow the guideline provided for you. You should not go below two reviews in you project. Don't forget to use inline comment while you are reviewing yourself Thanks . 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by aron-net** [comment]
Hi  @aron-helu,

While you made a great effort in this project, unfortunately, I can't proceed to review your code.

### Invalid Code Review Request  

-  You have submitted a project that has failing linters, Kindly make sure your linters are all passing before submitting the PR.

Cheers and Happy coding!

**Comment by aron-net** [comment]
### Hi  @aron-helu **:hand:,**

Your project is complete! There is nothing else to say other than... it's time to merge it.

Congratulations! :thumbsup:

Cheers and Happy coding! :clap: :clap: :clap:

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @aron-helu  ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 


## Required Changes ♻️

- [ ] It seems like you misunderstood the purpose of this exercise. The exercise is designed to get you to perform a review as if you were a TSE and to learn how to use certain GitHub features like inline comments. Please take a look at the [GItHub documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) in order to learn how to do so.
- [ ] You need to leave at least two inline comments suggesting changes in a polite manner. Please do so.

_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Zen+Loop&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css"> 
+    <title>Woldeab Woldemariam (ወልደአብ ወልደማርያም)</title>
+</head>
+<body>
+    <main id="main">
+        <h1 id="title">Woldeab Woldemariam (ወልደአብ ወልደማርያም)</h1>
+        <p>one of the original proponents of the Eritrean Independence movement </p>
```
Comment (aron-net) [inline index.html:16]: Kindly be noticed that the h1 and p tag to put in the header tag. The header element represents a container for introductory content or a set of navigational links.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            </ul>
+            <blockquote cite="http://news.rediff.com/report/2009/sep/14/pm-pays-tribute-to-father-of-green-revolution-borlaug.htm">
+                <p>"Up to now, I have never served nor become a messenger for any foreign power or interest. I am an enemy to any kind of slavery, in all its shapes and colors. No man, be he European or African, can force me into the yoke of any kind of bondage. If there be some one who dares to attempt my assassination in order to force me to submit to doing things contrary to my feeling and will, then I also have in me the courage to die for my political beliefs, for the cause of liberty of my country, and for the genuine interest of my brothers and sisters."</p>
+                <cite>-- Woldeab Woldemariam Quotes</cite>
+            </blockquote>
+            <h3>If you have time you should read more about this incredible human being on his 
+                <a id="tribute-link" rel="noopener" href="https://en.wikipedia.org/wiki/Woldeab_Woldemariam#cite_note-3" target="_blank">Wikipedia entry</a>.
+            </h3>
+        </section>
+    </main>
```
Comment (aron-net) [inline index.html:60]: Please add a footer to your page with contact information under the section tag.

**Review by aron-net — COMMENTED** [review]
### Required Changes :recycle:

Please check the comments under the review.

Submit for a re-review when you're done.

Thanks :smiley:

File: index.html (hunk 1 above)
Comment (zieeco) [reply index.html:16]: **[OPTIONAL]**

- Hey, you've done great with your review, although your explanation about the header tag is quite correct, however,  telling the code author to wrap the `h1` and `p` tag together in a `header` tag is contradicting your reasons regarding using the header tag. Based on your review, it'd been best if a `header` tag was added by the code author in an `article` like this: 

```
<article>
  <header>
    <h1>An Example</h1>
    <p>Revied on, <time datetime="2022-06-06">4 October 2017</time> by  Aron Alemseged</p>
  </header>
  <p>some more contents</p>
</article>
```

From the above example, it's okay to add the `p` there but not mandatory if the intention was just to wrap the `h1` in the `header` tag like this: 
```
<header>
  <h1>Main heading</h1>
</header>
```
Also, I think this is the main content on the page as used by the code author in the `main` tag, if peradventure the document was to have a header tag, then the code author would've added the `header` tag separately before the `main` tag.

> ***You've done well implementing the changes required by the previous reviewer.*** well-done :+1:

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @aron-helu 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Nice review comment 🔥 
✅ Good communication tone 👍 


## Optional suggestions
_check the comment under the reviews_

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**